integration/test/alertmanager/alertmanager.go

```diff
@@ -124,3 +64,65 @@ func GetAllAlertRules(dir string) ([]string, []string) {
 	}
 	return alertNames, exprList
 }
+
+func SendNotification(jsonData []byte) {
+	httpURL := Webhook
+	response, err := http.Post(httpURL, "", bytes.NewBuffer(jsonData))
+	utils.PanicIfNotNil(err)
+	log.Debug().Str("URL", httpURL).Bytes("json", jsonData).Msg("notification sent")
+
+	defer response.Body.Close()
+
+	if response.StatusCode == http.StatusOK {
+		_, err := ioutil.ReadAll(response.Body)
+		utils.PanicIfNotNil(err)
+	} else {
+		log.Error().Str("status", response.Status).Msg("Http post call failed with errors")
+	}
+}
+
+func processWebhook(writer http.ResponseWriter, request *http.Request) {
+	if request.URL.Path != WebhookEndpoint {
+		http.Error(writer, "404 page not found.", http.StatusNotFound)
+		return
+	}
+
+	switch request.Method {
+	case "POST":
+		webhookData := make(map[string]interface{})
```

Comment:
minor: but a struct would be nicer to work with here. I'll go over in meeting

Reply:
not needed as removed webhook code